libr/anal/xrefs.c

```diff
@@ -43,31 +47,32 @@ R_API bool r_anal_xrefs_load(RAnal *anal, const char *prjfile) {
 		if (!db) return false;
 		path = strdup (db);
 	} else {
-		db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
+		db = r_str_newf ("%s/%s.d", prjdir, prjfile);
+		db_hack = db + 2;
 		if (!db) return false;
-		path = r_str_home (db);
+		path = r_str_home (db_hack);
```

Comment:
r_str_abspath will resolve that '~' thing and other stuff like `..` and such, that what you must do instead of doing wrong +2 pointer increments. the same fix should go to the other PR